### PR TITLE
chore(fuselage): `MenuV2` custom button

### PR DESCRIPTION
**Pull request by juliajforesti**

### CODE REVIEW

File: packages/fuselage/src/components/Menu/V2/Menu.tsx [hunk 1]
```diff
@@ -28,6 +28,7 @@ interface MenuButtonProps<T> extends AriaMenuProps<T>, MenuTriggerProps {
   className?: ComponentProps<typeof Box>['className'];
   pressed?: boolean;
   maxWidth?: string;
+  customButton?: React.ReactElement;
```
Comment (tassoevan) [inline packages/fuselage/src/components/Menu/V2/Menu.tsx:31]: `nitpick` Could it be just `button`?

**Review by tassoevan — CHANGES_REQUESTED** [review]

File: packages/fuselage/src/components/Menu/V2/Menu.tsx (hunk 1 above)
Comment (juliajforesti) [reply packages/fuselage/src/components/Menu/V2/Menu.tsx:31]: I was wondering between these options:
1. menuButton
2. button
3. customButton
4. other?

imo `customButton` describes it more accurately, but i'm ok with `button` too. 
wdyt?

Comment (tassoevan) [reply packages/fuselage/src/components/Menu/V2/Menu.tsx:31]: Which would be the most familiar one for someone that knows Fuselage?

Comment (juliajforesti) [reply packages/fuselage/src/components/Menu/V2/Menu.tsx:31]: hummm, following the same reasoning as the `icon` being the option to chose a different one from the default (kebab), I agree with `button` :)

**Review by tassoevan — APPROVED** [review]

**Pull request merged**